File: Phan_A/bai31/bai_31.py
import sys

def main():
    input_data = sys.stdin.read().split()
    if not input_data:
        return
    n = int(input_data[0])
    a = [int(x) for x in input_data[1:n+1]]
    
    tong = sum(a)
    trung_binh = tong / n
    nho_nhat = min(a)
    lon_nhat = max(a)
    
    print(f"{tong} {trung_binh:.4f} {nho_nhat} {lon_nhat}")

if __name__ == "__main__":
    main() 

File: Phan_A/bai31/bai_31.jl
using Printf

function main()
    input_data = split(read(stdin, String))
    if isempty(input_data)
        return
    end
    n = parse(Int, input_data[1])
    a = [parse(Int, x) for x in input_data[2:n+1]]
    
    tong = sum(a)
    trung_binh = tong / n
    nho_nhat = minimum(a)
    lon_nhat = maximum(a)
    
    @printf("%d %.4f %d %d\n", tong, trung_binh, nho_nhat, lon_nhat)
end

main()